eng/deploy/azure/.env.example

```diff
@@ -1,2 +1,7 @@
 # The OAUTH_SIGNING_KEY may need quotation marks around the value, unlike other keys.
-OAUTH_SIGNING_KEY="<run `openssl rand -base64 256` to create a key>"
\ No newline at end of file
+OAUTH_SIGNING_KEY="<run `openssl rand -base64 256` to create a key>"
+
+# The following settings are values required for the URL, which is formed by {ED_FI_DOMAIN_NAME}.{AZURE_REGION}.azurecontainer.io
+# ED_FI_DOMAIN_NAME must be unique per Azure subscription
```

Comment:
Doesn't it need to be _globally_ unique, not just per subscription? Otherwise, two organizations could both have `meadowlark-abc.southcentralus.azurecontainer.io`.

Reply:
That's true